nodes/address_test.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nodes
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	kubernetesfake "k8s.io/client-go/kubernetes/fake"
+	"testing"
+)
+
+func TestNodesClientGuessNodeExternalAddress(t *testing.T) {
+	// given
```

Comment:
Why you don't like those comments? Is there some convention to not use those? I find it handy to see a test structure in that way.

Reply:
Most of the test follow the same structure: here's the input, act on it, check the result.
If it was out of ordinary then it would've made sense to comment.